### PR TITLE
Create MAINTAINERS.md

### PR DESCRIPTION
**Pull request by mit-mit**
Initial draft

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PR.
+
+See [CONTRIBUTING.md](https://github.com/grpc/grpc-community/blob/master/CONTRIBUTING.md)
+for general contribution guidelines.
+
+## Maintainers (in alphabetical order)
+- [mit-mit](https://github.com/mit-mit), Google Inc.
+- [nichite](https://github.com/nichite), Google Inc.
+- [szakarias](https://github.com/szakarias), Google Inc.
+- [sigurdm](https://github.com/sigurdm), Google Inc.
```
Comment (sigurdm) [inline MAINTAINERS.md]: @jonasfj @domesticmouse and @mkustermann each might or might not belong here.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Emeritus Maintainers (in alphabetical order)
+- [jakobr-google](https://github.com/jakobr-google), Google Inc.
```
Comment (sigurdm) [inline MAINTAINERS.md]: Do we need an emeritus list?
Earlier maintainers and contributors could belong in the AUTHORS file...

File: MAINTAINERS.md (hunk 1 above)
Comment (jonasfj) [reply MAINTAINERS.md]: I don't think I'm an active maintainer.

Comment (sigurdm) [reply MAINTAINERS.md]: I think this is actually a list of people with write-bits enabled.

Comment (mit-mit) [reply MAINTAINERS.md]: I'm not sure I see much reason to expand it beyond those that consider themselves maintainers

File: MAINTAINERS.md (hunk 2 above)
Comment (mit-mit) [reply MAINTAINERS.md]: That's what the open source team asked for. Apparently this is a common CNCF template

**Pull request merged**